cmd/flux/main_test.go

```diff
@@ -18,9 +18,10 @@ import (
 	fakeclient "sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
-func init() {
+func TestMain(m *testing.M) {
 	// Ensure tests print consistent timestamps regardless of timezone
-	os.Setenv("TZ", "UTC")
+	os.Setenv("TZ", "US")
```

Comment:
Let's do UTC please

Reply:
Ah, whoops, good catch, I set this to try to make sure it actually was still running to reproduce a test failure.

Fixed.